corehq/apps/toggle_ui/views.py

```diff
@@ -341,19 +341,30 @@ def _get_service_type(toggle):
     """Returns subscription service type for each toggle
     """
     service_type = {}
+    nested = defaultdict(lambda: defaultdict(list))
+    paused = defaultdict(list)
     for enabled in toggle.enabled_users:
         if _namespace_domain(enabled):
             domain = _enabled_item_name(enabled)
             if subscription := Subscription.get_active_subscription_by_domain(domain):
                 service_type[domain] = f"{subscription.service_type} : {subscription.plan_version.plan.name}"
+                if subscription.plan_version.plan.name.startswith("CommCare Paused Edition"):
+                    paused[(subscription.service_type, subscription.plan_version.plan.name)].append(domain)
+                else:
+                    nested[subscription.service_type][subscription.plan_version.plan.name].append(domain)
```

Comment:
I think you could just throw paused subscriptions in the same `nested` dict and avoid needing to duplicate the table in the template - something like:

```python
                if subscription.plan_version.is_paused:
                    nested["Paused"][service_type[domain]].append(domain)
                else:
                    nested[subscription.service_type][subscription.plan_version.plan.name].append(domain)
```

I dunno, maybe that's too non-homogeneous